### ISSUE TITLE
VS2015 "Error executing script" when typing import statement

### ISSUE THREAD

**Issue by nippur72** [Microsoft/TypeScript#11462]
**TypeScript Version:**   typescript@2.1.0-dev.20161008
**Visual Studio:** VS2015CE Update 3

**Code**

``` ts
import { Store } from ".    <--- caret is placed here
```

**Actual behavior:** 
When typing the dot `.` at line 1, the following error window appears in visual studio:
![error executing script](https://cloud.githubusercontent.com/assets/2598443/19213713/6add1aaa-8d73-11e6-9be9-7529fc292910.png)

**Comment by mhegazy** [comment]
Thanks for reporting this. the error window is shown because you have enabled [devMode](https://github.com/Microsoft/TypeScript/wiki/Dev-Mode-in-Visual-Studio) at some point. if you feel these errors are in your way, you can disable it and the errors will be reported silently without the popup.

**Comment by mhegazy** [comment]
looks related to completion in imports recent changes. @aozgaa can you take a look.

**Comment by aozgaa** [comment]
Hey @nippur72 , I'm having trouble reproducing this issue. Can you post a more detailed repro? In particular, what does the directory structure look like?

**Comment by nippur72** [comment]
@aozgaa opening the project on a different machine (same OS, VS and TS) does not give the error, so it must be something related to that particular machine, but I have no clue where to look for. Do you want me to try a fresh install of VS ?

Regarding the directory structure, the only strange thing is that `node_modules` is in the parent folder respect where the `.sln` / `.csproj` is:
```
project
+ node_modules
+ src
   + myproject.sln
   + mytsfiles.ts   
```

**Comment by aozgaa** [comment]
I wasn't able to reproduce the issue using commit  31a55e6  of TypeScript on VS 2015 Update 3 with TypeScript 2.0 Beta installed (I enabled dev mode and pointed VS towards my local build of TypeScript).

If the problem persists on the same machine after pulling the latest version of TS off master and building, let me know an I will look into this further.

**Comment by nippur72** [comment]
@aozgaa more clues:

If I use TS 2.0.3
```
npm install typescript@latest 
powershell .\VSDevMode.ps1 14 -tsScript node_modules/typescript/lib
```
there is no error.

But if I switch to nightly drops:
```
npm install typescript@next
powershell .\VSDevMode.ps1 14 -tsScript node_modules/typescript/lib
```
the error is back.

(The powershell command is [from here](https://github.com/Microsoft/TypeScript/wiki/Nightly-drops)).

**Comment by Methodician** [comment]
So just to clarify for those of us that want to develop without that error popping up every time we write an import statement, which is the appropriate course of action?

_remove the latest version of TS for Visual and install v2.0.3_ (worked)
       (OR)
_edit the registry to disable devMode (which I didn't knowingly enable at any time)_ (didn't try)

I did the easy one and switched back to version 2.0.3 which seems to have stopped the issue for now but I worry that I'll be missing out on some important update that occurred since then... and there's no guarantee that I'll be able to update upon the next versioning

**Comment by mhegazy** [comment]
> So just to clarify for those of us that want to develop without that error popping up every time we write an import statement, which is the appropriate course of action?

Switching off DevMod as [noted earlier](https://github.com/Microsoft/TypeScript/issues/11462#issuecomment-252678971) should make the pop up go away.

**Comment by aozgaa** [comment]
Still unable to repro. Would be happy to look into again this if you could link to a (minimal) reproducing project.
